src/polyfills.ts

```diff
@@ -42,7 +42,6 @@
 
 /** Evergreen browsers require these. **/
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
```

Comment:
Why is this removed?

Reply:
This is done automatically by `ng update`. 
It's because it should only be used in dev mode, but people weren't removing it for prod builds.

Reply:
Uh good to know!